scripts/README.md

```diff
@@ -108,4 +108,106 @@ CMD=update_database python3 scripts/003_vlan_type_string.py
 ```
 `update_database` changes the value of every outdated TAG from integer to their respective string value.
 
-</details>
\ No newline at end of file
+<details><summary><h3>Redeploy symmetric UNI vlans EVPLs </h3></summary>
+
+[`redeploy_evpls_same_vlans.py`](./redeploy_evpls_same_vlans.py) is a CLI script to list and redeploy symmetric (same vlan on both UNIs) EVPLs.
+
+You should use this script if you want to avoid a redundant `set_vlan` instruction that used to be present in the instruction set. This script by triggering an EVC redeploy will force that all flows get pushed and overwritten again, it'll temporarily create traffic disruption. The redeploy in this case is just to force that the flows are pushed right away instead of waiting for a network convergence that might result in the flows getting pushed again.
```

Comment:
it would be interesting here if you list which version would be impacted by this. For instance, if some deploys Kytos already on 2023.2, he/she does not have to worry about this, right?

Reply:
That's correct @italovalcy, added this info on https://github.com/kytos-ng/mef_eline/pull/390/commits/de5715f1853ba4e6d91e94c3755d2a53cc31b34c. Btw, in the future, we'll generally fix this general issue of knowing which (DB) scripts to run on https://github.com/kytos-ng/kytos-ng.github.io/issues/61, there's a general discussion going on there, if you also have any other general suggestions feel free to contribute there, thanks.